com.unity.render-pipelines.universal/Shaders/LitMetaPass.hlsl

```diff
@@ -3,46 +3,24 @@
 
 #include "Packages/com.unity.render-pipelines.universal/ShaderLibrary/MetaInput.hlsl"
 
-struct Attributes
-{
-    float4 positionOS   : POSITION;
-    float3 normalOS     : NORMAL;
-    float2 uv0          : TEXCOORD0;
-    float2 uv1          : TEXCOORD1;
-    float2 uv2          : TEXCOORD2;
-#ifdef _TANGENT_TO_WORLD
-    float4 tangentOS     : TANGENT;
-#endif
-};
-
-struct Varyings
-{
-    float4 positionCS   : SV_POSITION;
-    float2 uv           : TEXCOORD0;
-};
-
-Varyings UniversalVertexMeta(Attributes input)
-{
-    Varyings output;
-    output.positionCS = MetaVertexPosition(input.positionOS, input.uv1, input.uv2, unity_LightmapST, unity_DynamicLightmapST);
-    output.uv = TRANSFORM_TEX(input.uv0, _BaseMap);
-    return output;
-}
-
-half4 UniversalFragmentMeta(Varyings input) : SV_Target
+half4 UniversalFragmentMetaLit(MetaVaryings input) : SV_Target
```

Comment:
Not sure I follow why renaming this and why needed to rename `Varyings` to `MetaVaryings`.